city/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String
```

Comment:
Ensure that the 'City' model is correctly defined with all necessary fields as per the task description. This includes 'id', 'name', and 'additional_info'.